internal/commands/user/inputs.go

```diff
@@ -71,10 +71,10 @@ func (i *usersInputs) ResolveUsers(ui terminal.UI, client realm.Client, app real
 	if usersErr != nil {
 		return nil, usersErr
 	}
-	if len(i.Users) > 0 {
-		if len(foundUsers) == 0 {
-			return nil, errors.New("no users found")
-		}
+	if len(i.Users) > 0 && len(foundUsers) == 0 {
+		return nil, errors.New("no users found")
+	}
+	if len(i.Users) > 0 || len(foundUsers) == 0 {
```

Comment:
I modified the logic here to prevent an unexpected error case when an app has no users. I tried some different refactoring but this one worked the best for both manual and unit tests